scripts/india_nhm/districts/nhm_birth_control/NHM_BirthControl.mcf

```diff
@@ -0,0 +1,63 @@
+Node: dcid:Count_BirthControlEvent_Vasectomy
+description: "Number of Vasectomies Conducted (Public and Private Institutions)"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_Vasectomy
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthControlEvent_Tubectomy
+description: "Number of Tubectomies Conducted (Public and Private Institutions)"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_Tubectomy
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthControlEvent_Sterilization
+description: "Total Sterilisation Conducted"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_Sterilization
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthControlEvent_Vasectomy_AsFractionOf_Count_BirthControlEvent_Sterlization
+description: "Percent of Male Sterlisation (Vasectomies) to Total sterilisation"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_Vasectomy_AsFractionOf_Count_BirthControlEvent_Sterlization
+statType: dcs:measuredValue
+
+Node: dcid:Count_Death_BirthControlSterilization
+description: "Total cases of deaths following Sterilisation (Male and Female)"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_Death_BirthControlSterilization
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthControlEvent_IUCDInsertion
+description: "Total IUCD Insertions done"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_IUCDInsertion
+statType: dcs:measuredValue
+
+Node: dcid:Count_BirthControlEvent_IUCDInsertion_AsFractionOf_Count_BirthControlEvent
+description: "Percent of IUCD insertions to all family planning methods"
+typeOf: dcs:StatisticalVariable
+populationType: schema:Person
+measuredProperty: dcid:Count_BirthControlEvent_IUCDInsertion_AsFractionOf_Count_BirthControlEvent
+statType: dcs:measuredValue
+
+Node: dcid:Count_ContraceptiveDistribution_OralPill
+description: "Oral Pills distributed"
```

Comment:
This will be number of pills distributed to the public hospitals, which is actually the count of pills.

We can change the description to "Number of distributed contraceptives (oral pills)"